letstest/README.md

```diff
@@ -19,35 +19,23 @@ This package is installed in the Certbot development environment that is
 created by following the instructions at
 https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client.
 
-After activating that virtual environment, you can then configure AWS
-credentials and create a key by running:
-```
->aws configure --profile <profile name>
-[interactive: enter secrets for IAM role]
->aws ec2 create-key-pair --profile <profile name> --key-name <key name> --query 'KeyMaterial' --output text > whatever/path/you/want.pem
-```
-Note: whatever you pick for `<key name>` will be shown to other users with AWS access.
+These tests use the AWS SDK for Python (boto3) to manipulate EC2 instances.
+Before running the tests, you'll need to set up credentials by following the
+instructions at
+https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration.
+You will also want to create a `~/.aws/config` file setting the region for your
+profile to `us-east-1`.
```

Comment:
```suggestion
profile to `us-east-1`, following the instructions in the boto3 quickstart guide above.
```

Just to make it clearer for someone who hasn't clicked the link yet or stops when they get to the credentials instructions, because otherwise it looks like we don't have instructions for that part.